### PR TITLE
Site: make closing CLI-terminal more visible

### PR DESCRIPTION
**Pull request by layakdev**

### CODE REVIEW

File: site/content/en/docs/start/_index.md [hunk 1]
```diff
@@ -455,7 +455,8 @@ choco install minikube
       [Environment]::SetEnvironmentVariable('Path', $('{0};C:\minikube' -f $oldPath), [EnvironmentVariableTarget]::Machine) `
     }
     ```
-    _If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._
+    <span style="color:blue">
+    _If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding._</span>
```
Comment (medyagh) [inline site/content/en/docs/start/_index.md:459]: Thank you for this PR, 
it seems like after changing this PR the "_" no longer makes the font "italic"

we could do replace "_" with "*" 

"If you used a CLI to perform the installation, you will need to close that CLI and open a new one before proceeding."

**Review by medyagh — CHANGES_REQUESTED** [review]

File: site/content/en/docs/start/_index.md (hunk 1 above)
Comment (medyagh) [reply site/content/en/docs/start/_index.md:459]: also how about we change the wording to
"If you used a terminal (like powershell) for the installation, you will need to close the terminal and reopen it before running minikube."

**Comment by layakdev** [comment]
@medyagh 
Hi, ready to review =)
Thanks

**Review by medyagh — APPROVED** [review]
Looks good to me ! thank you for this PR, look forward to see more contributions from you !

**Pull request merged**